### PR TITLE
Simplify examples

### PR DESCRIPTION
**Pull request by utku-caglayan**
Some examples may be lengthy for command line interface (small window size)

### CODE REVIEW

File: types/mapcmd/get.go [hunk 1]
```diff
@@ -25,8 +25,7 @@ import (
 )
 
 const MapGetExample = `  # Get value of the given key from the map.
-  hzc map get -n mapname -k k1
-  hzc map get --key hello --name myMap
+  hzc map get --key hello --name myMap   # default key-type is string
```
Comment (yuce) [inline types/mapcmd/get.go:28]: Do you think we can add the `key-type` here and get rid of the example below?

File: types/mapcmd/put.go [hunk 2]
```diff
@@ -26,35 +26,11 @@ import (
 	"github.com/hazelcast/hazelcast-commandline-client/internal"
 )
 
-const MapPutExample = `  # Put key, value pair to map.
-  hzc put -n mapname -k k1 -v v1
+const MapPutExample = `  # Put key, value pair to map. ttl/max-idle can be (ns,us,ms,s,m,h)
```
Comment (yuce) [inline types/mapcmd/put.go:29]: How about "The unit for ttl/max-idle is one of (ns,us,ms,s,m,h)"

File: types/mapcmd/put.go [hunk 3]
```diff
@@ -26,35 +26,11 @@ import (
...
+  map put --key-type string --key hello --value-type float32 --value 19.94 --name myMap --ttl 3ms --max-idle 4ms
```
Comment (yuce) [inline types/mapcmd/put.go]: How about using `s` and `m` for TTL and max idle? I think TTL < 1 is not legal.

**Review by yuce — COMMENTED** [review]
Some ideas...

File: types/mapcmd/get.go (hunk 1 above)
Comment (utku-caglayan) [reply types/mapcmd/get.go:28]: How about just keeping the example below?

**Review by alisengul53 — APPROVED** [review]

**Pull request merged**